Content.Server/Weapon/Melee/MeleeWeaponSystem.cs

```diff
@@ -125,7 +133,7 @@ private void OnClickAttack(EntityUid owner, MeleeWeaponComponent comp, ClickAtta
 
             if (args.Target is {Valid: true} target)
             {
-                // Raise event before doing damage so we can cancel damage if the event is handled
+                // Raising a melee hit event which may handle combat for us
```

Comment:
pretty sure this event was only ever raised on the item to get damage. isnt it useless now? i think this event should have just been renamed, not a new one added

Reply:
Its used in Flashes and Zombies amongst other things to handle other effects than dealing damage.